src/main/java/com/openshift/internal/restclient/model/ImageStream.java

```diff
@@ -44,6 +45,16 @@ public ImageStream(ModelNode node, IClient client, Map<String, String []> proper
 		super(node, client, propertyKeys);
 		this.propertyKeys = propertyKeys;
 	}
+	
+	@Override
+	public DockerImageURI getDockerImageRepositorySpec() {
```

Comment:
why not call this #getDockerImageURI to match what you get back when you call it?